code/script.js

```diff
@@ -1,47 +1,160 @@
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+  const name = nameInput.value;
+  userAnswer(`${name}`);
+  // This will keep the input box empty after user clicks send
+
+  nameInput.value = "";
+  setTimeout(() => serviceOptions(name), 1000);
+});
+
+// 2nd Bot message
+const serviceOptions = (name) => {
+  questionStep++;
+  botAnswer(`Hi ${name} 👋 What can we help you with today?`);
+  inputWrapper.innerHTML = `
+    <button id="plumbingButton"> Plumbing 🔧 </button>
+    <button id="electricalButton"> Electrical Work 🔌 </button>
+    <button id="paintingButton"> Painting 🖌️ </button>
+    `;
+  document.getElementById("plumbingButton").addEventListener("click", () => {
+    type = "plumbing";
+    handleInput("plumbing");
+  });
+  document.getElementById("electricalButton").addEventListener("click", () => {
+    type = "electrical";
+    handleInput("electrical");
+  });
+  document.getElementById("paintingButton").addEventListener("click", () => {
+    type = "painting";
+    handleInput("painting");
+  });
+};
+
+//3rd Bot message
+const moreServices = (type) => {
+  questionStep++;
+  botAnswer(
+    `So you need help with ${type}. Which type of ${type} service will you require?`
+  );
+  // Here we added a conditional statement for specific service options depending on the category.
+  if (type === "painting") {
+    inputWrapper.innerHTML = `
+  <select id = 'select'>
+  <option value='' selected disabled> Choose service </option>
+  <option value='interior painting'>Interrior Painting</option>
+  <option value='exterior painting'>Exterior Painting</option>
+  <option value='wall resurfacing'>Wall Resurfacing</option>
+  </select> `;
+  } else {
+    inputWrapper.innerHTML = `
+    <select id = 'select'>
+    <option value='' selected disabled> Choose service </option>
+    <option value='installation'>Installation</option>
+    <option value='maintenece'>Maintenece</option>
+    <option value='repair'>Repair</option>
+    <option value='emergency repair'>Emergency Repair</option>`;
+  }
+  const select = document.getElementById("select");
+  select.addEventListener("change", () => handleInput(select.value));
+};
+
+//4th Bot message
+const priceInformation = (service) => {
+  questionStep++;
+  if (service === "emergency repair") {
+    botAnswer(`Perfect! Our starting price for ${service} is SEK 5,000.`);
+  } else {
+    botAnswer(`Perfect! Our starting price for ${service} is SEK 3,000.`);
+  }
+
+  inputWrapper.innerHTML = `
+<button id="bookButton">I want to book</button>
+`;
+  document
+    .getElementById("bookButton")
+    .addEventListener("click", () => handleInput("I want to book"));
+};
+
+// 5 th bot message
+function goodBye() {
+  botAnswer(
+    `Thank you for booking ${type} with us, some will contact you soon. `
+  );
+  botAnswer(`Have a lovley day!`);
+  // Below clears the option to click book after once selected.
+  inputWrapper.innerHTML = ``;
+}

```

Comment:
if you'd like to refresh the page when all chat is ended, you can add `window.location.reload(); ` here as our project did!